packages/ffe-file-upload/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@sb1/ffe-file-upload",
+  "version": "1.0.0",
+  "description": "File upload module",
+  "license": "MIT",
+  "author": "SpareBank 1",
+  "main": "less/ffe-file-upload.less",
+  "repository": {
+    "type": "git",
+    "url": "ssh://git@github.com:SpareBank1/designsystem.git"
+  },
+  "scripts": {
+    "lint": "stylelint less/*.less",
+    "test": "npm run lint"
+  },
+  "devDependencies": {
+    "@sb1/ffe-core": "^14.0.4",
```

Comment:
This is now mismatching your peer dependency. I would suggest perhaps accepting a wider range of versions in peer dependencies.
Perhaps `>=14.0.0` or `14.x || 15.x`?

Reply:
Regarding the file structure, it's by design the files you upload should not contain duplicates, however, the logic enforcing this currently rest in the implementation, which is not ideal.

In the example, if you upload a file with same name, you overwrite the old one.

Reply:
Okay. That is fine by me, just wanted to make sure we are aware of it, and that we've made a conscious decision that it's fine this way :)